packages/manager/apps/zimbra/public/translations/accounts/Messages_fr_FR.json

```diff
@@ -12,5 +12,11 @@
   "zimbra_account_tooltip_need_slot": "Aucun compte disponible, veuillez en commander.",
   "zimbra_account_delete_modal_content_step1": "Vous êtes sur le point de supprimer le compte email du serveur Zimbra.",
   "zimbra_account_delete_modal_content_step2": "Êtes-vous sûr de vouloir supprimer la boite mail du serveur Zimbra ?",
-  "zimbra_account_delete_modal_warn_message": "Attention cette action est irréversible et entraîne la suppression des données."
+  "zimbra_account_delete_modal_warn_message": "Attention cette action est irréversible et entraîne la suppression des données.",
+  "zimbra_account_upgrade_cta_back": "Retour vers mes comptes emails",
+  "zimbra_account_upgrade_new_offer": "Nous lançons en version bêta notre <Link href=\"https://www.ovhcloud.com/fr/emails/zimbra-emails/\">nouvelle offre Zimbra Pro</Link>.",
+  "zimbra_account_upgrade_offer_details": "Nous vous proposons de mettre à niveau votre compte gratuitement pendant toute la durée de la phase bêta.",
+  "zimbra_account_upgrade_lab_page": "Pour plus de détails, veuillez lire <Link href=\"https://labs.ovhcloud.com/en/zimbra-beta/\">les conditions de béta sur la page lab</Link>.",
```

Comment:
Even if is only one english url, it's better to put the url outside the translation